.github/in-cluster-test-scripts/external-workloads.sh

```diff
@@ -4,7 +4,7 @@ set -x
 set -e
 
 # Run connectivity test
-cilium connectivity test --debug --all-flows
+cilium connectivity test --debug --all-flows --test '!host-entity.*'
```

Comment:
Can you document why these tests are being skipped in the code? If there is an outstanding issue that we need to solve in order to run these tests in this environment, can you file an issue or link it here?

Reply:
I have added remote-note in policy, so that we don't need to skip this test.